lib/student_greeting_checker.rb

```diff
@@ -30,12 +30,13 @@ def initialize(student, wiki, greeters)
       @student = student
       @wiki = wiki
       @all_greeters = greeters
+      @username = greeters.pluck(:username)
```

Comment:
This is an array of possibly many usernames, so the variable name should not be singular.

Reply:
This `Check` class gets initialized once for every student being checked, so I don't think this solves the problem; it will still run a new pluck query for every student checked.

Reply:
ah, gotcha - I misunderstood your instance variable ask. It should be an instance variable on `StudentGreetingChecker`. Could then pass that as a param to `Check`. Would that work instead?